server/src/main/java/io/spine/examples/kanban/server/column/ColumnAdditionProcess.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2022, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.kanban.server.column;
+
+import io.spine.examples.kanban.ColumnAddition;
+import io.spine.examples.kanban.ColumnId;
+import io.spine.examples.kanban.command.CreateColumn;
+import io.spine.examples.kanban.command.PlaceColumn;
+import io.spine.examples.kanban.event.ColumnAdded;
+import io.spine.examples.kanban.event.ColumnAdditionRequested;
+import io.spine.examples.kanban.event.ColumnCreated;
+import io.spine.examples.kanban.event.ColumnPlaced;
+import io.spine.server.command.Command;
+import io.spine.server.event.React;
+import io.spine.server.procman.ProcessManager;
+
+/**
+ * Coordinates addition of a new column to a board.
+ */
+public final class ColumnAdditionProcess
+        extends ProcessManager<ColumnId, ColumnAddition, ColumnAddition.Builder> {
+
+    /**
+     * Whenever a request to add a new column is made, issue a command to
```

Comment:
We answer to questions "what this thing is?" or "what it does". 

So I would start with "Issues ... whenever ...". Notice the third form of the first verb here. 

Same comment applies to the code below. There is also a formatting issue like the one I pointed out previously.